pkg/motor/login.go

```diff
@@ -161,6 +169,33 @@ func createWalletConfigs(id string, req mt.LoginWithKeysRequest, shards vault.Va
 		return nil, fmt.Errorf("psk shard: %s", err)
 	}
 
+	bankShard0, err := mpc.AesDecryptWithKey(req.AesPskKey, shards.ShardBank[0])
```

Comment:
Decrypting all of these bank shards shouldn't be necessary. Providing just the two from above is enough to create the wallet.